contracts/ccip/GovernanceCCIPRelay.sol

```diff
@@ -32,33 +31,86 @@ contract GovernanceCCIPRelay is IGovernanceCCIPRelay {
   /// @dev Constructor to initialize the GovernanceRelay contract.
   /// @param _timelock The address of the Timelock contract.
   /// @param _router The address of the CCIP router contract.
-  /// @param _destinationChainSelector The chain selector for the destination chain.
-  /// @param _receiver The address of the receiver contract on the destination chain.
+  /// @param _destinationChainSelectors Array of chain selectors for the destination chains.
+  /// @param _destinationReceivers Array addresses of the receiver contracts on the destination chains.
   constructor(
     address _timelock,
     address _router,
-    uint64 _destinationChainSelector,
-    address _receiver
+    uint64[] memory _destinationChainSelectors,
+    address[] memory _destinationReceivers
   ) {
     ccipRouter = IRouterClient(_router);
     timelock = _timelock;
-    destinationChainSelector = _destinationChainSelector;
-    destinationReceiver = _receiver;
+    _addDestinationChains(_destinationChainSelectors, _destinationReceivers);
   }
 
   /// @inheritdoc IGovernanceCCIPRelay
-  function setDestinationReceiver(address _receiver) external onlyTimeLock {
-    emit DestinationReceiverUpdated(destinationReceiver, _receiver);
-    destinationReceiver = _receiver;
+  function addDestinationChains(
+    uint64[] memory _destinationChainSelectors,
+    address[] memory _destinationReceivers
+  ) external onlyTimeLock {
+    _addDestinationChains(_destinationChainSelectors, _destinationReceivers);
+  }
+
+  function _addDestinationChains(
+    uint64[] memory _destinationChainSelectors,
+    address[] memory _destinationReceivers
+  ) private {
+    uint256 receiversLength = _destinationReceivers.length;
+    require(
+      _destinationChainSelectors.length == receiversLength,
+      ArrayLengthMismatch()
+    );
+
+    for (uint256 i = 0; i < receiversLength; ) {
+      uint64 _destinationChainSelector = _destinationChainSelectors[i];
+      address _destinationReceiver = _destinationReceivers[i];
+
+      require(
+        ccipRouter.isChainSupported(_destinationChainSelector),
+        DestinationChainNotSupported(_destinationChainSelector)
+      );
+
+      require(
+        _destinationChainSelector != CCIPMainnetChainSelector,
+        CannotUseMainnetChainSelector()
+      );
+
+      require(
+        _destinationReceiver != address(0),
+        ReceiverCannotBeZeroAddress()
+      );
+
+      require(
+        destinationReceivers[_destinationChainSelector] == address(0),
```

Comment:
@voith you don't have a way to update/remove a `(chain, receiver)` while you could do that before.

What if you want to change the receiver for a chain or remove the support to the chain?

Reply:
I added `updateDestinationReceiver` to handle this scenario 👍